src/composables/useClient.ts

```diff
@@ -55,43 +57,62 @@ export function useClient() {
   }
 
   async function sendEIP712(space, type, payload) {
-    if (type === 'proposal') {
-      let plugins = {};
-      if (Object.keys(payload.metadata?.plugins).length !== 0)
-        plugins = payload.metadata.plugins;
-      return clientEIP712.proposal(auth.web3, web3.value.account, {
-        space: space.id,
-        type: payload.type,
-        title: payload.name,
-        body: payload.body,
-        discussion: payload.discussion,
-        choices: payload.choices,
-        start: payload.start,
-        end: payload.end,
-        snapshot: payload.snapshot,
-        plugins: JSON.stringify(plugins),
-        app: 'snapshot'
-      });
-    } else if (type === 'vote') {
-      return clientEIP712.vote(auth.web3, web3.value.account, {
-        space: space.id,
-        proposal: payload.proposal.id,
-        type: payload.proposal.type,
-        choice: payload.choice,
-        privacy: payload.privacy,
-        app: 'snapshot',
-        reason: payload.reason
-      });
-    } else if (type === 'delete-proposal') {
-      return clientEIP712.cancelProposal(auth.web3, web3.value.account, {
-        space: space.id,
-        proposal: payload.proposal.id
-      });
-    } else if (type === 'settings') {
-      return clientEIP712.space(auth.web3, web3.value.account, {
-        space: space.id,
-        settings: JSON.stringify(payload)
-      });
+    const aliased = space.voting.aliased;
+    try {
+      if (aliased) {
+        await checkAlias();
+        if (!aliasWallet.value || !isValidAlias.value) await setAlias();
+      }
+
+      const [provider, address] = aliased
+        ? [aliasWallet.value, aliasWallet.value.address]
+        : [auth.web3, web3.value.account];
+
```

Comment:
@bonustrack this conditional is setting `provider` and `address` based on the `aliased` setting. I used it below for `vote`, `proposal` and `delete-proposal`. Settings still use `auth.web3` and `web3.value.account` regardless of `aliased`

Reply:
Looks good!

Reply:
And this is the update you asked about above:
> @zzuziak I don't see the update or comment you are referring to here anywhere. Could you link it?